projects/novo-elements/src/elements/picker/extras/mixed-multi-picker-results/MixedMultiPickerResults.ts

```diff
@@ -166,6 +167,13 @@ export class MixedMultiPickerResults extends BasePickerResults implements OnDest
         return !!(primaryOption && primaryOption.showSearchOnSecondaryOptions);
     }
 
+    public resetPrimaryOption(primaryOption: IMixedMultiPickerOption) {
+        if (this.internalMap.get(primaryOption.value)) {
+            this.internalMap.delete(primaryOption.value);
+            this.selectPrimaryOption(primaryOption);
```

Comment:
Do we always want this option to be selected whenever the BehaviorSubject emits an event? I could see use cases in the future where we want to clear the cached secondary options of several primary options, and in that case we wouldn't want to select them all after